pvsystemprofiler/tilt_azimuth_study.py

```diff
@@ -39,6 +37,8 @@ def __init__(self, data_handler, day_range=None, init_values=[10, 10],
             print('Running DataHandler preprocessing pipeline with defaults')
             self.data_handler.run_pipeline()
         self.init_values = init_values
```

Comment:
The following construction makes use of the preferred `is None` pattern (see: http://jaredgrubb.blogspot.com/2009/04/python-is-none-vs-none.html) and avoids an extra variable assignment.

`if init_values is None:
    self.init_values = [10, 10]
else:
    self.init_values = init_values`